server/src/modules/Question/question.service.ts

```diff
@@ -39,26 +39,18 @@ export class QuestionService {
         }
     }
 
-    async insert(question: QuestionDto): Promise<QuestionDto> {
+    async insert(question: QuestionDto): Promise<Question> {
```

Comment:
I think here should be QuestionDto.

Reply:
changed for consistency purposes for whole file